### PR TITLE
Fix issue with goecoding

### PR DESCRIPTION
**Pull request by aniansson**
## Pivotal Tracker story 

https://www.pivotaltracker.com/story/show/

## Changes proposed in this pull request:

We had an issue getting the session to work on everyones computer. The server kept on complaining on that the ```lat = session[:geo_location]['lat']``` had to be changed to ```lat = session[:geo_location].lat``` and the other way around. Therefore we put in an else and if method, for now. It was a blocker for everyone working with the code -so we needed a fast solution.  

Also deleted method act_as_mappable due to unnecessary code that is not being used in the user model. 

## What I have learned working on this feature:

Else and if methods are helpful.

### CODE REVIEW

**Comment by AmberWilkie** [comment]
As I mentioned, this fix won't be right until there's a test to expose the underlying problem behavior. I realize that this is going to give people problems when they pull from upstream, but someone should figure out how to expose it. I suggest this falls on @Philippoes because he implemented the code, but that's up to you guys to decide as a group.

**Comment by aniansson** [comment]
It's just a temporary solution cause the others have to keep on working, since it was a big blocker. So we will decide on who will solve the issue and make it an issue on pivotal.

File: app/models/user.rb [hunk 1]
```diff
@@ -4,8 +4,5 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
   has_many :orders
-
-  acts_as_mappable lat_column_name: :lat,
-                   lng_column_name: :lng,
-                   default_units: :kms
+
```
Comment (AmberWilkie) [inline app/models/user.rb]: Is this change related to the error?
